### PR TITLE
Size to bytes

### PR DESCRIPTION
**Pull request by sergeymirzoyan1991**

### CODE REVIEW

File: src/adapters/gcp/clients/gcp-disks-client.ts [hunk 1]
```diff
@@ -100,7 +101,7 @@ export class GcpDisksClient extends GcpBaseClient implements GcpClientInterface,
             v.name,
             StringHelper.splitAndGetAtIndex(v.zone, '/', -1) || '',
             StringHelper.splitAndGetAtIndex(v.type, '/', -1) || '',
-            (parseFloat(v.sizeGb) | 0) * 1073741824,
+            SizeHelper.gbToByte(parseFloat(v.sizeGb) | 0),
```
Comment (marashlyansuren) [inline src/adapters/gcp/clients/gcp-disks-client.ts:104]: should not this be `||`

File: src/adapters/gcp/gcp-price-calculator.ts [hunk 2]
```diff
@@ -68,8 +69,9 @@ export class GcpPriceCalculator {
       const ramPrice = skus.filter((db) => db.key === ramKey && db.regions?.includes(item.getRegion()))[0]?.price
       const cpuPrice = skus.filter((db) => db.key === cpuKey && db.regions?.includes(item.getRegion()))[0]?.price
       const storagePrice = skus.filter((db) => db.key === storageKey && db.regions?.includes(item.getRegion()))[0]?.price
+      console.log(ramPrice , item.ram , cpuPrice , item.cpu , storagePrice , item.storage)
```
Comment (marashlyansuren) [inline src/adapters/gcp/gcp-price-calculator.ts]: Remove this please

File: src/adapters/gcp/gcp-price-calculator.ts [hunk 3]
```diff
@@ -68,8 +69,9 @@ export class GcpPriceCalculator {
...
       if (ramPrice && item.ram && cpuPrice && item.cpu && storagePrice && item.storage) {
-        item.pricePerMonth = ramPrice * item.ram / 1000 + cpuPrice * item.cpu + storagePrice * item.storage
+        item.pricePerMonth = ramPrice * SizeHelper.byteToGb(item.ram) / 1000 + cpuPrice * item.cpu + storagePrice * SizeHelper.byteToGb(item.storage)
```
Comment (marashlyansuren) [inline src/adapters/gcp/gcp-price-calculator.ts:73]: why are we touching to price logic

**Review by marashlyansuren — APPROVED** [review]

File: src/adapters/gcp/gcp-price-calculator.ts [hunk 4]
```diff
@@ -69,7 +70,7 @@ export class GcpPriceCalculator {
       const cpuPrice = skus.filter((db) => db.key === cpuKey && db.regions?.includes(item.getRegion()))[0]?.price
       const storagePrice = skus.filter((db) => db.key === storageKey && db.regions?.includes(item.getRegion()))[0]?.price
       if (ramPrice && item.ram && cpuPrice && item.cpu && storagePrice && item.storage) {
-        item.pricePerMonth = ramPrice * item.ram / 1000 + cpuPrice * item.cpu + storagePrice * item.storage
+        item.pricePerMonth = ramPrice * SizeHelper.byteToMb(item.ram) / 1000 + cpuPrice * item.cpu + storagePrice * SizeHelper.byteToGb(item.storage)
```
Comment (noraarakelyan) [inline src/adapters/gcp/gcp-price-calculator.ts:73]: @sergeymirzoyan1991 jan seems like gcps ram, disk size are also calculated in gib
https://cloud.google.com/compute/disks-image-pricing#:~:text=For%20Compute%20Engine%2C%20disk%20size,on%20Cloud%20Platform%20SKUs%20apply.

File: src/adapters/gcp/clients/gcp-sql-client.ts [hunk 5]
```diff
@@ -126,8 +127,8 @@ export class GcpSqlClient extends GcpBaseClient implements GcpClientInterface {
         v.databaseVersion,
         'secondaryGceZone' in v,
         parseFloat(StringHelper.splitAndGetAtIndex(v.settings.tier, '-', -2) || '0'),
-        parseFloat(StringHelper.splitAndGetAtIndex(v.settings.tier, '-', -1) || '0'),
-        v.settings.dataDiskSizeGb,
+        SizeHelper.mbToByte(parseFloat(StringHelper.splitAndGetAtIndex(v.settings.tier, '-', -1) || '0')),
+        SizeHelper.gbToByte(v.settings.dataDiskSizeGb || 0),
         v.createTime,
```
Comment (noraarakelyan) [inline src/adapters/gcp/clients/gcp-sql-client.ts:132]: @sergeymirzoyan1991 jan seems like gcps ram, disk size are also calculated in gib
https://cloud.google.com/compute/disks-image-pricing#:~:text=For%20Compute%20Engine%2C%20disk%20size,on%20Cloud%20Platform%20SKUs%20apply.

**Pull request merged**